ads/inabox/position-observer.js

```diff
@@ -25,6 +26,8 @@ import {Observable} from '../../src/observable';
  */
 let PositionEntryDef;
 
+/** @const */
+const MIN_EVENT_INTERVAL_IN_MS = 100;
```

Comment:
Isn't that clearer?
Otherwise, how do you know it's interval about space or time :-)